lib/index.js

```diff
@@ -469,6 +471,40 @@ class AuthenticationPlugin {
     return this.getUsersRepository().delete(user._id, {refresh: 'wait_for'});
   }
 
+  /**
+   * Search users by ES query concerning exclusively their username
+   *
+   * @param {Object} query
+   *
+   * @returns {Promise.<{ scrollId, hits, aggregations, total }>}
+   */
+  async search (query) {
+    debug('exists(query = %j)', query);
+
+    const sanitizedQuery = objectRecursiveMap(query, (key) => {
```

Comment:
Nitpicking: coding standards
```suggestion
    const sanitizedQuery = objectRecursiveMap(query, key => {
```